### PR TITLE
Create add reducers and actions

### PR DESCRIPTION
**Pull request by Lembani**
>Highlights:
- Add `React Redux` (using: $ npm install redux react-redux).
- Use a "feature folder" approach and the [ducks pattern](https://github.com/erikras/ducks-modular-redux) for my Redux files and app files.
- Add the book's `actions and reducer` (in /src/redux/books/books.js) using the [ducks pattern](https://github.com/erikras/ducks-modular-redux).
- Add categories `actions and reducer` (in /src/redux/categories/categories.js) using the [ducks pattern](https://github.com/erikras/ducks-modular-redux).
- Configure the `Redux Store` (/src/redux/configureStore.js)
- Other minor improvements.

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+const ADD_BOOK = 'bookstore/src/redux/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookstore/src/redux/books/REMOVE_BOOK';
```
Comment (Kingstalux) [inline src/redux/books/books.js]: - [ ] Good job so far 👍 The correct way of writing the action type names according to the [ducks pattern](https://github.com/erikras/ducks-modular-redux) is `const ADD_BOOK = 'bookstore/books/ADD_BOOK';` Kindly correct all your action type names 👍

File: src/redux/categories/categories.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+const CHECK_STATUS = 'bookstore/src/redux/categories/CHECK_STATUS';
```
Comment (Kingstalux) [inline src/redux/categories/categories.js]: - [ ] Same issue here. Kindly write this action type value using the correct format 👍

File: src/components/AddBook.js [hunk 3]
```diff
@@ -1,16 +1,38 @@
...
+      case 'author':
+        setData({ author: e.target.value, book: data.book });
+        break;
+      default:
+        setData({ book: data.book, author: data.author });
+    }
+  };
+
+  const dataStore = () => {
+    console.log(data);
```
Comment (Kingstalux) [inline src/components/AddBook.js]: - [ ] Kindly remove all console statements so as to keep your code clean 💯

**Review by Kingstalux — CHANGES_REQUESTED** [review]
Hi @Lembani, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f744b797263304e546c4563475764546870342f67697068792e676966


## Highlights 👏 
✅ Good PR description 🔥 
✅ Professional README file 💯 
✅ Linters are passing 👌 
✅ Good commit history 👍 



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/books/books.js (hunk 1 above)
Comment (Lembani) [reply src/redux/books/books.js]: Noted thank you. Let me do it right away.

File: src/redux/books/books.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
+const ADD_BOOK = 'bookstore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
```
Comment (mateo951) [inline src/redux/books/books.js:2]: Addressed ✔

**Review by mateo951 — APPROVED** [review]
### Hi @Lembani, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🤓 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/books/books.js (hunk 4 above)
Comment (Lembani) [reply src/redux/books/books.js:2]: Thanks a lot for approving my PR. 🙏🏾

**Pull request merged**